docs/methods/properties/bandstructure_gw.md

```diff
@@ -206,8 +205,8 @@ experimental ionization potential (12.62 eV). Benchmarks on molecules and solids
 self-consistency of eigenvalues in the Green's function G improves the agreement between the GW
 calculation and experiment. This scheme is called GW0@PBE and is our favorite GW flavor so far
 (experience based on nano-sized aromatic molecules). You can also check
-[this video](https://www.youtube.com/watch?v=1vUuethWhbs&t=5563s) or the [GW Compendium] on which
-DFT starting functional and which self-consistency scheme could be good for your system.
+[this video](https://www.youtube.com/watch?v=1vUuethWhbs&t=5563s) or [Golze2019](#Golze2019) on
```

Comment:
You can embed videos like this:

````
```{youtube} 1vUuethWhbs
---
url_parameters: ?start=5563
align: center
privacy_mode:
---
```
````